### DISCUSSION TITLE
连接阿里云Hologres，数据库配置不能正常识别

### DISCUSSION THREAD

**Discussion by VictorZan** [hhyo/Archery#2278] [Q&A]
### 重现步骤

按照pgsql配置方式连接阿里云Hologres

### 预期外的结果

1.配置的数据库信息不能正常识别，默认以用户名当数据库。
2.不配置数据库信息，默认以用户名当数据库。

### 日志文本

```shell
暂无
```

### 版本

1.9.1

### 部署方式

Docker

### 是否还有其他可以辅助定位问题的信息？比如数据库版本等

暂无

**Comment by LeoQuote** [comment]
先升到 master 再试试

**Reply by VictorZan** [reply]
是master
